src/com/company/Main.java

```diff
@@ -3,6 +3,113 @@
+
+        int targetCommon = 7; // кг
+        int targetPerDay1 = 250; // грамм
+        int targetPerDay2 = 500; // грамм
+
+        int targetInGramms = targetCommon * 1000;
+        int nDays1 = targetInGramms / targetPerDay1;
+        int nDaysPart1 = targetInGramms % targetPerDay1;
+        int nDays2 = targetInGramms / targetPerDay2;
+        int nDaysPart2 = targetInGramms % targetPerDay2;
+
+        System.out.println("Количество дней в первом варианте: " + nDays1 + "." + nDaysPart1);
+        System.out.println("Количество дней во втором варианте: " + nDays2 + "." + nDaysPart2);
+
+        int nAverage = (nDays1 + nDays2) / 2;
+        System.out.println("Количество дней в среднем: " + nAverage + "\n");
+
+        // Task 5
+
+        int mashaSalary = 67_760;
+        int denisSalary = 83_690;
+        int kristinaSalary = 76_230;
+
+        int mashaSalaryPerYear = 12 * mashaSalary;
+        int denisSalaryPerYear = 12 * denisSalary;
+        int kristinaSalaryPerYear = 12 * kristinaSalary;
+
+        System.out.println("Маша получала " + mashaSalaryPerYear + " руб. в год");
+        System.out.println("Денис получал " + denisSalaryPerYear + " руб. в год");
+        System.out.println("Кристина получала " + kristinaSalaryPerYear + " руб. в год");
+
+        float increaseCount = 10.0f / 100.0f;
+        float mashasIncrease = mashaSalary * increaseCount;
+        float denisesIncrease = denisSalary * increaseCount;
+        float kristinasIncrease = kristinaSalary * increaseCount;
+
+//        System.out.println("Прибавка к зарплате Маше " + mashasIncrease + " руб.");
+//        System.out.println("Прибавка к зарплате Денису " + denisIncrease + " руб.");
+//        System.out.println("Прибавка к зарплате Кристине " + kristinaIncrease + " руб.");
+
+        mashaSalary += mashasIncrease;
+        denisSalary += denisesIncrease;
+        kristinaSalary += kristinasIncrease;
+
+        System.out.println("Теперь Маша получает " + mashaSalary + " руб. в месяц");
+        System.out.println("Теперь Денис получает " + denisSalary + " руб. в месяц");
+        System.out.println("Теперь Кристина получает " + kristinaSalary + " руб. в месяц");
+
+        int mashaSalaryPerYearNew = 12 * mashaSalary;
+        int denisSalaryPerYearNew = 12 * denisSalary;
+        int kristinaSalaryPerYearNew = 12 * kristinaSalary;
+
+        int mashasDifference = mashaSalaryPerYearNew - mashaSalaryPerYear;
+        int denisesDifference = denisSalaryPerYearNew - denisSalaryPerYear;
+        int kristinasDifference = kristinaSalaryPerYearNew - kristinaSalaryPerYear;
+
+        System.out.println("Годовой доход Маши возрос на " + mashasDifference + " руб.");
+        System.out.println("Годовой доход Дениса возрос на " + denisesDifference + " руб.");
+        System.out.println("Годовой доход Кристины возрос на " + kristinasDifference + " руб.");
+
+        System.err.println("Итог: Ну как бы у всех годовой доход возрос на 10%, все верно :))");
```

Comment:
```suggestion
        System.out.println("Итог: Ну как бы у всех годовой доход возрос на 10%, все верно :))");
```
err лучше использовать для вывода ошибок, как и гласит его название.

Reply:
Ну, я так сделал, чтобы этот вывод выделялся среди остальных, т.к. он к теме особо не относится :)